rails/planit/config/routes.rb

```diff
@@ -3,5 +3,6 @@
   ActiveAdmin.routes(self)
   root "home#index"
   get "/:category", to: "home#index"
+  resources :providers
```

Comment:
this enables the create,new,index,delete,update routes but in the controller you only defined index and show. Get used to only defining the routes that you need:
`resources :providers, only: [:index, :show]`